cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -960,7 +962,7 @@ def download(self, req):
         def _get_origin_format(path):
             qcow2_length = 0x9007
             if path.startswith('http://') or path.startswith('https://') or path.startswith('ftp://'):
-                resp = urllib2.urlopen(path)
+                resp = urllib.request.urlopen(path)
```

Ruff (0.15.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)